### ISSUE TITLE
Suggestion: Unused Variables

### ISSUE THREAD

**Issue by DanielGoehler** [markbrummel/vscode-allint#17]
I know there is a reference indicator above a variables, but you only seen it, if you right in front of that variable. Would be nice to be able the analyze the hole solution and list all unused variables.

**Comment by markbrummel** [comment]
I agree. Analyzing the whole solution will be somewhat of a winter project because I want to display the results in a nice way.

If you look at the TypeScript there is an object called summary. This is what I am planing on creating and send as json to a webpage or excel.

Keep the great ideas comming please.

**Comment by markbrummel** [comment]
Started to work on it. I have a basic version working that detects unused variables. Added two new objects. alLine and alCodePart. Still playing with the latter. Should get it to work I guess.
